tests/scripts/thread-cert/backbone/bbr_5_11_01.py

```diff
@@ -91,6 +91,9 @@ def test(self):
         self.simulator.go(5)
         self.assertEqual('router', self.nodes[ROUTER2].get_state())
 
+        # The OTBR docker enables SRP Server by default, lets explicitly
+        # disable SRP server to avoid Network Data population.
```

Comment:
should we disable SRP server by default in OTBR docker?

Reply:
I think the SRP server should be auto-started in OTBR when the feature is compiled.

Actually, I would expect our test scripts can tolerate additional Service TLV in network data and we should not need the change here.